site/gatsby-site/src/components/TaxonomyForm.js

```diff
@@ -362,6 +362,15 @@ const EditTaxonomyForm = ({
       if (f.display_type === 'date') {
         newValues[f.short_name] = `${values[f.short_name]}T00:00:00.000Z`;
       }
+
+      //Convert string into boolean
+      if (f.display_type === 'bool') {
+        if (values[f.short_name] === '') {
+          newValues[f.short_name] = undefined;
```

Comment:
@smcgregor if an empty value is set for boolean fields - unsettling - it will add undefined. this line dictates it